renderer.md

```diff
@@ -0,0 +1,30 @@
+## Disclaimer
+
+First of all this is an attempt to gather information about state of Renderer in dlang
+
+# What is a Renderer?
+ 
+Renderer contains a rendering state and provides functionality to draw primitives using specific color or texture. Renderer uses Image as a buffer to render to. Calling renderer method issues command to underlying hardware. The command is queued and flushed at some later time using some form of `flush` method. Renderer draw the following primitives:
```

Comment:
> The command is queued and flushed at some later time ~using some form of `flush` method~.

I don't think there's any reason to specify the mechanism. I doubt a user would want to call `flush` very often though. Likely just let it render in its own time.

> Calling renderer method issues command to ~underlying hardware~ the backend API.

Maybe this?